src/pages/sponsors.astro

```diff
@@ -0,0 +1,296 @@
+    link: "https://bolt.new/",
+  },
+  {
+    name: "clerk",
+    image: "/assets/sponsors/clerk.svg",
+    description: "The easiest way to add authentication to your react app.",
+    link: "https://clerk.com/",
+  },
+  {
+    name: "browserbase",
+    image: "/assets/sponsors/browserbase.svg",
+    description: "Web browser for your ai. Like puppeteer, but better.",
+    link: "https://www.browserbase.com/",
+  },
+  {
+    name: "unkey",
+    image: "/assets/sponsors/unkey.svg",
+    description:
+      "Unkey is an open-source API key management platform that allows you to create, manage, and monitor API keys for your applications.",
+    link: "https://unkey.dev/",
+  },
+
+  {
+    name: "frontend masters",
+    image: "/assets/sponsors/frontend-masters.svg",
+    description:
+      "The best place to learn modern frontend development. Frontend Masters is a platform for learning modern frontend development with React, Vue, and Svelte.",
+    link: "https://www.frontendmasters.com/",
+  },
+  {
+    name: "payload",
+    image: "/assets/sponsors/payload.svg",
+    description:
+      "The best headless CMS for React, especially if you're using Next.js.",
+    link: "https://www.payloadcms.com/",
+  },
+  {
+    name: "singlestore",
+    image: "/assets/sponsors/singlestore.svg",
+    description:
+      "The only database you'll ever need. Scale from one to one million customers, handling SQL, JSON, full text and vector workloads — all in one unified platform.",
+    link: "https://www.singlestore.com/",
+  },
+  {
+    name: "fal",
+    image: "/assets/sponsors/fal.svg",
+    description: "Generative media platform for developers.",
+    link: "https://www.fal.ai/",
+  },
+  {
+    name: "ragie ai",
+    image: "/assets/sponsors/ragie.svg",
+    description:
+      "RAG-as-a-Service for developers. Fully managed, fully connected.",
+    link: "https://www.ragie.ai/",
+  },
+  {
+    name: "agora",
+    image: "/assets/sponsors/agora.png",
+    description:
+      "Conversational AI · Video · Voice · Chat: Flexible APIs supported by the only global network engineered for live interaction.",
+    link: "https://www.agora.io/",
+  },
+  {
+    name: "code crafters",
+    image: "/assets/sponsors/code-crafters.svg",
+    description:
+      "The best place to learn modern frontend development. Frontend Masters is a platform for learning modern frontend development with React, Vue, and Svelte.",
+    link: "https://codecrafters.io/",
+  },
+  {
+    name: "bright data",
+    image: "/assets/sponsors/bright-data.svg",
+    description:
+      "The best proxy network for developers. Bright Data is a proxy network that allows you to scrape the web, crawl the web, and access the web.",
+    link: "https://www.brightdata.com/",
+  },
+  {
+    name: "eight sleep",
+    image: "/assets/sponsors/eight-sleep.svg",
+    description: "Turn any bed into the ultimate sleeping experience",
+    link: "https://www.eightsleep.com/",
```

Comment:
Make eight sleep a separate section on top for "affiliates". Add SuperHuman as well.